packages/transaction-controller/src/TransactionController.ts

```diff
@@ -159,6 +159,14 @@ export type TransactionControllerMessenger = RestrictedControllerMessenger<
   never
 >;
 
+/**
+ * A function for verifying a transaction, whether it is malicious or not
+ */
+export type SecurityProviderRequest = (
```

Comment:
Shall we move this to `types.ts`?

Reply:
Fixed in https://github.com/MetaMask/core/pull/1725/commits/de01ef1b344c1d7fba67652f68639fe73c886324